api/src/auth.js

```diff
@@ -83,12 +84,12 @@ module.exports = {
    * @return     {Object}  {username: username, password: password}
    */
   basicAuthCredentials: req => {
-    const authHeader = req && req.headers && req.headers.authorization;
+    const authHeader = req?.headers?.authorization;
     if (!authHeader || !authHeader.startsWith('Basic ')) {
       return false;
     }
     try {
-      const [username, password] = Buffer.from(authHeader.split(' ')[1], 'base64').toString().split(':');
+      const [username, password] = atob(authHeader.split(' ')[1]).split(':');
```

Comment:
nitpick: why change this?  Looks like `atob` is [marked as "legacy"](https://nodejs.org/api/all.html#all_buffer_bufferatobdata) in favor of `Buffer.from`...

Reply:
Oh, my bad. I'll switch it back!